kolibri/core/theme_hook.py

```diff
@@ -37,6 +37,7 @@
 APP_BAR = "appBar"
 BACKGROUND = "background"
 TITLE = "title"
+TITLE_STYLE = "titleStyle"
```

Comment:
Just to make sure I understand - this is optional, because it's not listed in the validation that happens in the frontend?

Reply:
which validation are you referring to?

Everything in the theme hook is supposed to be optional, though the handling somewhat case-specific. In this case, `:style="$theme.signIn.titleStyle"` will just be `undefined`

Reply:
Oh, I think it's just the brand colors that are required? https://github.com/learningequality/kolibri/blob/release-v0.12.y/kolibri/core/assets/src/styles/theme.js#L68

Reply:
ah that's right, forgot.

I consolidated validation in the theme hook:

https://github.com/learningequality/kolibri/blob/aea553cc4ce3b98b424ddb92ae4e5a182e911609/kolibri/core/theme_hook.py#L66-L96

Reply:
Ah - ok. I just remember the issue I had on the frontend where I had to make a JS copy of the theme for the frontend tests :)

Reply:
right - that test theme could have been marginally smaller

Reply:
added a follow-up issue: https://github.com/learningequality/kolibri/issues/5689